components/decorateMapComponent.js

```diff
@@ -48,7 +48,7 @@ export default function decorateMapComponent(Component, { componentType, provide
     if (components[provider]) return components[provider];
 
     if (provider === PROVIDER_DEFAULT) {
-      components.default = getDefaultComponent();
+      if (!components.default) components.default = getDefaultComponent();
```

Comment:
thanks for catching this! I wonder if it would be better to do:

```
components[DEFAULT_PROVIDER] = getDefaultComponent();
```

instead.... It's just a little weird because the value of `DEFAULT_PROVIDER` is `null`.